dashboard/src/pages/ingest/batches/[id]/index.vue

```diff
@@ -0,0 +1,153 @@
+<script setup lang="ts">
+import Tooltip from "bootstrap/js/dist/tooltip";
+import { onMounted, ref } from "vue";
+
+import { api } from "@/client";
+import StatusBadge from "@/components/StatusBadge.vue";
+import StatusLegend from "@/components/StatusLegend.vue";
+import UUID from "@/components/UUID.vue";
+import uploader from "@/composables/uploader";
+import { useAuthStore } from "@/stores/auth";
+import { useBatchStore } from "@/stores/batch";
+import IconInfo from "~icons/akar-icons/info-fill";
+
+const authStore = useAuthStore();
+const batchStore = useBatchStore();
+
+const el = ref<HTMLElement | null>(null);
+let tooltip: Tooltip | null = null;
+
+let showLegend = ref(false);
```

Comment:
We're only mutating the inner value, this should be a const, right?

Reply:
Fixed in other pages too.